newsfragments/921.feature

```diff
@@ -0,0 +1,3 @@
+Moved `dials.export format=best` to `dials.export_best` as that one needed 
```

Comment:
As I understand this is intended to form the release notes - so could it be made a more user-friendly description of the change? I think just moved to standalone dials.export_best command would suffice, the user doesn't need to know about the underlying reasons for the change in this case.

Reply:
I think we can tidy that up when we get to it, but I would leave in some of the details as _some_ end users know about the environment variable nonsense so making it clear that this is no longer needed would be useful. However if you disagree feel free to revise.